### ISSUE TITLE
Connect to Kusto Explorer

### ISSUE THREAD

**Issue by tikkanu** [MicrosoftDocs/dataexplorer-docs#845]
Hi Team,

It is mentioned on this page that "You can run the queries using client tools that support Kusto queries, such as: Kusto Explorer, Azure Data Explorer Web UI, Jupyter Kqlmagic, Flow, PowerQuery, PowerShell, Lens, REST API."

Could you please explain how exactly would someone connect a Kusto Explorer to an Azure Log Analytics Workspace?

Thank you!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6f6c8fe3-d9a3-f11b-cfe7-f0f67f44b0d5
* Version Independent ID: 418e6ed4-24b4-689b-50fc-490441dc393c
* Content: [Query data in Azure Monitor with Azure Data Explorer](https://docs.microsoft.com/en-us/azure/data-explorer/query-monitor-data)
* Content Source: [data-explorer/query-monitor-data.md](https://github.com/MicrosoftDocs/dataexplorer-docs/blob/master/data-explorer/query-monitor-data.md)
* Service: **data-explorer**
* GitHub Login: @osalzberg
* Microsoft Alias: **orspodek**

**Comment by osalzberg** [comment]
Hi,

you can add a workspace just like any kusto cluster that you add, use the URL that is in the doc as the cluster URL

As I wrote in the doc,
In the Add Cluster window, add the URL of the LA or AI cluster.

In the Add Cluster window, add the URL of the LA or AI cluster.
For LA: https://ade.loganalytics.io/subscriptions/<subscription-id>/resourcegroups/<resource-group-name>/providers/microsoft.operationalinsights/workspaces/<workspace-name>
For AI: https://ade.applicationinsights.io/subscriptions/<subscription-id>/resourcegroups/<resource-group-name>/providers/microsoft.insights/components/<ai-app-name>

**Comment by osalzberg** [comment]
I can see the the URL broke in the comment, use the ones that are in the [doc](aka.ms/adxproxy)

**Comment by tikkanu** [comment]
Thank you for the response. I have tried this, and am getting the following error:

"Failed to connect to the cluster. Do you want to add this connection anyway?

Connection attempt resulted with the following error:
	Endpoint 'https://ade.loganalytics.io' is not responsive. This could indicate that the service in unavailable or does not exist"

NsLookup from my machine, resolves the host:

Non-authoritative answer:
Name:    azmon-prod-weu-0-ingress-adx-proxy.westeurope.cloudapp.azure.com
Address:  51.138.53.92
Aliases:  ade.loganalytics.io
          adxproxyla.trafficmanager.net

And from dataexplorer.azure.com I can add the same cluster URL without a problem.

Any other suggestions?

Thank you in advance.

**Comment by osalzberg** [comment]
Hi,
I have just retried it in my lab environment. I have opened the latest version of Kusto Explorer, clicked on the Add connection button and copied and pasted `https://ade.loganalytics.io/subscriptions/<subscription-id>/resourcegroups/<resource-group-name>/providers/microsoft.operationalinsights/workspaces/<workspace-name>`, changed anything between the <> marks and clicked connect. it works.

Can you send me a screenshot of the issue and the URL that you are trying to add to my email? orens@microsoft.com

**Comment by tikkanu** [comment]
Hi Oren,

Sure. Will send it right away. Thanks!

**Comment by tikkanu** [comment]
@YaelSchuster yes, it has been switch to direct email, due to sensitive nature of the data shared, and I am waiting for an update from @osalzberg.

**Comment by YaelSchuster** [comment]
@tikkanu ok thanks, so I'll let you two figure that out. @osalzberg please let me know if there are docs updates as a result. 
At this time, I'm going to close this issue since you're otherwise in communication.
#please-close

**Comment by tikkanu** [comment]
Hey Oren,

Here is the screenshot i have also sent over email:

![kusto (002)](https://user-images.githubusercontent.com/31651387/138223684-753172a4-d062-4c78-a6b7-09c674dcfb81.png)

Thank you!

**Comment by tikkanu** [comment]
Hi @osalzberg ,

Do you have any news on this topic?

Thanks in advance!

**Comment by osalzberg** [comment]
Hi,

Sorry but not really. Im still trying to figure out who owns the tool

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by tikkanu** [comment]
No problem, Oren. I was just checking in, so to speak. Will wait patiently :)

Thank you!
